### PR TITLE
Vipa suite theme

### PR DESCRIPTION
**Pull request by tylerstokes**

### CODE REVIEW

File: Stylesheets/src/Neptune/Pages/Support/page--support.scss [hunk 1]
```diff
@@ -747,9 +757,11 @@ div.note {
 
 		.note_content > div,
 		.file-upload .multi-file-upload {
-			margin: 5px 0;
+			margin-top: 5px;
 			margin-left: 30px;
-			width: 80%;
+		}
+		.file-upload {
+		    margin-top: 10px;
```
Comment (sbenesventuretech) [inline Stylesheets/src/Neptune/Pages/Support/page--support.scss]: Why is this there? this would cause alignment issues

File: Stylesheets/src/Neptune/Pages/Support/page--support.scss [hunk 2]
```diff
@@ -747,9 +757,11 @@ div.note {
 
 		.note_content > div,
 		.file-upload .multi-file-upload {
-			margin: 5px 0;
+			margin-top: 5px;
```
Comment (sbenesventuretech) [inline Stylesheets/src/Neptune/Pages/Support/page--support.scss]: why are you switching this to top only?

File: Stylesheets/src/Neptune/Pages/Support/page--support.scss [hunk 3]
```diff
@@ -724,13 +724,22 @@ div.note {
 				overflow-y: auto;
 			}
 		}
+		div.field_SUMMARY,
+		div.field_THIRDPARTYID2 {
+		    margin-right: 30px;
```
Comment (sbenesventuretech) [inline Stylesheets/src/Neptune/Pages/Support/page--support.scss]: why is this added? This shouldn't be needed at all.

File: Stylesheets/src/Neptune/Pages/Support/page--support.scss (hunk 1 above)
Comment (tylerstokes) [inline Stylesheets/src/Neptune/Pages/Support/page--support.scss]: @sbenesventuretech It is just giving spacing from the optional file description as it was too close to it. I don't see any alignment issues with this...

File: Stylesheets/src/Neptune/Pages/Support/page--support.scss (hunk 3 above)
Comment (tylerstokes) [inline Stylesheets/src/Neptune/Pages/Support/page--support.scss]: @sbenesventuretech This is needed so that the input under both of these doesn't change when re-sizing the window.

File: Stylesheets/src/Neptune/Pages/Support/page--support.scss (hunk 2 above)
Comment (tylerstokes) [inline Stylesheets/src/Neptune/Pages/Support/page--support.scss]: @sbenesventuretech This was a mistake. I can change it back to margin: 5px 0; It is needed.

**Comment by proteusklogan** [comment]
Closing due to being really old PR and it seems to not be needed.
